config.json

```diff
@@ -1425,6 +1425,17 @@
         "trees"
       ]
     },
+    {
+      "slug": "darts",
+      "uuid": "cb581e2c-66ab-4221-9884-44bacb7c4ebe",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 2,
+      "topics": [
+          "algorithms",
+          "control_flow"
```

Comment:
Topics should be taken from the [global topics list](https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt); `control_flow` is not listed there.

I'm not sure `algorithms` can be applied here.

What about:
```JSON
"topics": [
	"conditionals",
	"math"
]
```

Reply:
I agree and have made changes.